src/components/include/cos_kernel_api.h

```diff
@@ -107,5 +107,63 @@ int cos_hw_detach(hwcap_t hwc, hwid_t hwid);
 void *cos_hw_map(struct cos_compinfo *ci, hwcap_t hwc, paddr_t pa, unsigned int len);
 int cos_hw_cycles_per_usec(hwcap_t hwc);
 
+/* Default API */
+/* Capabilities for Async activation end point */
+struct cos_aep_info {
+	tcap_t    tc;
+	thdcap_t  thd;
+	arcvcap_t rcv;
+};
+
+/* Default Component information */
+struct cos_defci {
+	struct cos_compinfo ci;
+	struct cos_aep_info sched_aep;
+};
+
+/* current component's default component information */
+extern struct cos_defci curr_defci;
```

Comment:
If this is statically defined in all components that use the `cos_kernel_api`, then we might want to make it in a separate library/`.c` file so that we don't waste the memory for it in components that don't use it.